navigation to homepage done

Dulya opened the pull request:
Page navigation for the issue #1, I've solved this to get a basic idea on how the development goes.
Used navigator for both android and ios.

Dulya committed:
navigation to homepage done

berlintam commented:
wow, thanks so much for this @Dulya !

@starkcoffee can you take the lead on checking this over? (or maybe we can go over it on Monday)

starkcoffee commented:
Cool! I can review on weekend and summarize to you Tam?

On 10 Mar 2017 10:29 a.m., "Tam" <notifications@github.com> wrote:

> wow, thanks so much for this @Dulya <https://github.com/Dulya> !
>
> @starkcoffee <https://github.com/starkcoffee> can you take the lead on
> checking this over? (or maybe we can go over it on Monday)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sounddrop/sounddrop-react/pull/3#issuecomment-285620641>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAlACN8LAtO7R4e9dgoK-rETtV9I_Fzdks5rkRfngaJpZM4MZIor>
> .
>

starkcoffee commented on index.android.js:
```diff
@@ -22,6 +23,8 @@ var SoundDrop = React.createClass({
 		switch (route.id) {
       case 'welcome':
         return (<Main navigator={navigator}/>);
+	  case 'home':
```
Just need to fix the indentation here.

starkcoffee commented:
Thanks for figuring out how to do navigation! I wonder about the additional constructor() and navigate() functions in the Main component. I notice that the Main component uses `onPress={()=>this.navigate('home')}` and the Home component uses `onPress={()=>this.props.navigator.pop()}`. One of them uses a convenience function and one of them doesn't. It would be good to be consistent.

Dulya committed:
indentation fixed and comments added to main component

Dulya committed:
indentation fix to index.android.js

Dulya committed:
removed navigate function and constructor of main component to make it consistent

Dulya commented:
I've fixed the indentation issue. It took me a while to figure out the consistency issue. Now I've fixed that by removing constructor and navigate functions, so that a single push line now handles the navigation in the render function.
I'm still learning react native, and thank you so much for the help and pointing out the issues in the code.

starkcoffee commented:
Woops, I need to check my github notifications, I didn't see this was updated. Thanks a lot!
